README.md

```diff
@@ -22,76 +22,44 @@ The following procedure is to be followed each new semester by the maintainer fo
-  Enter your password and security answer when prompted.
-
-  Your timetable will be saved in `data.txt`. Make any changes required in `data.txt`. Then, proceed to Step 2.
+- Clone this repository 
+- Install the dependencies by running:
+    ```sh
+    pip install -r requirements.txt
+    ```
+- Run the gyft script:
+    ```sh
+    python3 gyft.py
+    ```
+  - Optional flags: 
+    - `--input` or `i`: Input file (`.txt`) containing the timetable data/to which timetable data will be saved. **Default:** `data.txt` 
+    - `--output` or `o`: Output file (`.ics`) to which the ICS file will be written. **Default:** `timetable.ics`
+    - `--del-events`: To delete events automatically added by the script before adding new events (explained later) **Default:** `False`
+
+- Enter your roll number, password, security answer and OTP (if required) when prompted
+
+- Your timetable will be saved in the file specified by the `--input` flag (**default:** `data.txt`)
+- Now you can choose between 2 options, which will be prompted to you:
+- #### Add to Google Calendar:
+	- For this, you have to get your `credentials.json` and save it as `client_secret.json` to the current directory by following the Step 1 from [here](https://developers.google.com/google-apps/calendar/quickstart/python#step_1_turn_on_the_api_name)
+	- Also note that adding to Google Calendar requires an Internet connection
   
-  **Use the web application on any browser**
-
-  - Go to the [GYFT WebApp](https://gyft.metakgp.org/).
-  - Enter your roll number and get the security question.
-  - Once the security question is fetched, enter your credentials and save the ICS file.
-  - Move to step 2(b)(ii).
+- #### Generate ICS file:
+  - This will generate an ICS file which you can import into any calendar application.
+  - For importing to Google Calendar, follow the instructions given [here](https://support.google.com/calendar/answer/37118?hl=en).
   
-  Source at [metakgp/gyft-serve](https://github.com/metakgp/gyft-serve).
-
-- **Step 2:** Decide whether you want to add the events to Google Calendar or
-    generate an ICS file from the data.
-
-    Adding to Google Calendar requires an Internet connection
-
-    ICS files are compatible with almost all Calendar applications (including
-    the iOS calendar application, Sunrise etc)
-
-- **Step 2(a):** If you decide on adding your events to your Google Calendar:
-
-    - **Step (i):** Get your `credentials.json` and save it as `client_secret.json` to the current directory by
-                    following the Step 1 from
-                    [here](https://developers.google.com/google-apps/calendar/quickstart/python#step_1_turn_on_the_api_name).
-
-    - **Step (ii):** Now, run:
-
-        ```sh
-        $ python3 add_events.py
-        ```
-
-    - **MISC:** To delete all the recurring events added by the CLI or GYFT Web Application (uses property of recurrence to identify events to delete):
-
-        ```sh
-        $ python3 del_events.py
```

Comment:
These steps should be kept since `add_events.py` and `del_events.py` work perfectly fine.